### PR TITLE
update rounding rule for % calculations; COUNTRY=rbd

### PR DESCRIPTION
**Pull request by wadhwamatic**
### Description

Update CD data for RBD to match rounding rules used in official reporting

## Checklist - did you ...

Test your changes with

- [ ] `REACT_APP_COUNTRY=rbd yarn start`
- [ ] `REACT_APP_COUNTRY=cambodia yarn start`
- [ ] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: frontend/public/data/rbd/ch/al_admin2_2020_10_01.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
             "adm0_name": "Niger",
             "adm0_pcod2": "NE",
             "adm0_pcod3": "NER",
+            "adm0_name.1": "Niger",
+            "adm0_pcod2.1": "NE",
+            "adm0_pcod3.1": "NER",
```
Comment (ericboucher) [inline frontend/public/data/rbd/ch/al_admin2_2020_10_01.json:9]: is this change expected?

**Review by ericboucher — APPROVED** [review]

File: frontend/public/data/rbd/ch/al_admin2_2020_10_01.json (hunk 1 above)
Comment (wadhwamatic) [reply frontend/public/data/rbd/ch/al_admin2_2020_10_01.json:9]: weird. I'm not sure how that got introduced. will look into it.

Comment (ericboucher) [reply frontend/public/data/rbd/ch/al_admin2_2020_10_01.json:9]: on a more general note, the CH files are quite big, so it's nice if we can limit the data to what we actually need to limit their size and impact on performance

Comment (wadhwamatic) [reply frontend/public/data/rbd/ch/al_admin2_2020_10_01.json:9]: Yes, we can trim data. I kept all fields for now even though they're not all being used as it's still being fleshed out a bit

**Comment by wadhwamatic** [comment]
@ericboucher I created an issue to address file size. Will get to that early April

**Pull request merged**